tests/testthat/test-build-home-citation.R

```diff
@@ -29,6 +29,15 @@ test_that("source link is added to citation page", {
   expect_true(any(grepl("<code>inst/CITATION</code></a></small>", lines)))
 })
 
+test_that("citation page includes inst/AUTHORS", {
+  pkg <- local_pkgdown_site(test_path("assets/inst-authors"))
+  suppressMessages(init_site(pkg))
```

Comment:
#2330 will be nice :sweat_smile:

Reply:
Btw I'm not convinced it will be, because I think there are two meanings of `quiet` currently.